_component_design/buttons.md

```diff
@@ -4,9 +4,9 @@ parent: Editing
 secondary: Inputs and Controls
 layout: component
 category: Components
-usage: |
-  Use primary, secondary and tertiary button types to signal important actions.
-preview-image: components/preview-selectors.svg
+usage: >
+  Buttons direct user attention to important actions that can be performed from the current page. Varies types of buttons are used based on the importance of each action.
```

Comment:
`Varies` to `Various`. Also, I'm guessing this page still needs info dev review as I think I remember the text here was just pulled from the content of the page itself, right? If not, I'd change the last sentence to active voice:
```
Base the use of various types of buttons on the importance of each action.
```

Reply:
Fixed in commit fe6a7cf and added to this PR. Also changed the last sentence to active voice, although this page will still go through an info dev cycle upcoming.